.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "javascript.validate.enable": false
+}
```

Comment:
```suggestion
```

Reply:
i add this for remove vscode error : 

<img width="844" alt="Capture d’écran 2024-07-22 à 00 13 43" src="https://github.com/user-attachments/assets/59f0e8b2-ed92-4a72-810c-4a9381b43e37">

it's recommended in setup documentation of the plugin https://github.com/flow/flow-for-vscode?tab=readme-ov-file#setup